.github/workflows/conformance.yaml

```diff
@@ -53,7 +53,9 @@ jobs:
           - cleanup
           - events
           - exceptions
-          - generate
+          - generate/clusterpolicy
+          - generate/policy
+          - generate/validation
```

Comment:
no `generate/e2e` ?

Reply:
I moved it to clusterpolicy/standard, it's a test for the subresource.